tests/pkg_semtech-loramac/README.md

```diff
@@ -0,0 +1,150 @@
+## Semtech LoRaMAC package test application
+
+### About
+
+This is a test application for the Semtech LoRaMAC package. This package
+provides the MAC primitives for sending and receiving data to/from a
+LoRaWAN network.
+
+See [LoRa Alliance](https://www.lora-alliance.org/) for more information on LoRa.
+See [Semtech LoRamac-node repository](https://github.com/Lora-net/LoRaMac-node)
+to have a look at the original package code.
+
+This application can only be used with Semtech
+[SX1272](http://www.semtech.com/images/datasheet/sx1272.pdf) or
+[SX1276](http://www.semtech.com/images/datasheet/sx1276.pdf) radio devices.
+
+### Application configuration
+
+Before building the application and joining a LoRaWAN network, you need an
+account on a LoRaWAN backend provider. Then create a LoRaWAN application and
+register your device.
+Since this application has been heavily tested with the backend provided by
+[TheThingsNetwork](https://www.thethingsnetwork.org/) (TTN), we recommend that
+you use this one.
+
+Once your application and device are created and registered, you'll have
+several information (provided by the LoRaWAN provider):
+* The type of join procedure: ABP (Activation by personnalization) or OTAA (Over
+The Air Activation)
+* The device EUI: an 8 bytes array
+* The application EUI: an 8 bytes array
+* The application key: a 16 bytes array
+* The device address: a 4 bytes array, only required with ABP join procedure
+* The application session key: a 16 bytes array, only required with ABP join procedure
+* The network session key: a 16 bytes array, only required with ABP join procedure
+
+Once you have this information, either edit the `Makefile` accordingly or
+use the `set`/`get` commands in test application shell.
+
+### Building the application
+
+The default parameters for the Semtech SX1272/SX1276 radios works as-is with
+ST Nucleo 64 boards and MBED LoRa shields
+([SX1276](https://os.mbed.com/components/SX1276MB1xAS/) or
+[SX1272](https://os.mbed.com/components/SX1272MB2xAS/)). You can also use the
+ST [b-l072z-lrwan1](http://www.st.com/en/evaluation-tools/b-l072z-lrwan1.html)
+board.
+
+Depending on the type of radio device, set the `DRIVER` variable accordingly:
+For example:
+```
+    DRIVER=sx1272 make BOARD=nucleo-f411 -C pkg/semtech-loramac flash term
```

Comment:
I tried this but it failed !

Reply:
ok, I forgot to rename one internal variable, fixed.